### DISCUSSION TITLE
browsable api link is not working , giving application error

### DISCUSSION THREAD

**Discussion by Krishna2864** [encode/django-rest-framework#9172] [General]
HI everyone 
the web browsable api link is not working on homepage 
```
here Application error
An error occurred in the application and your page could not be served. If you are the application owner, [check your logs for details](https://devcenter.heroku.com/articles/logging#view-logs). You can do this from the Heroku CLI with the command
heroku logs --tail
```
![Uploading Screenshot from 2023-11-29 18-55-28.png…]()

**Comment by lovelydinosaur** [comment]
Yep, if someone wants to issue a PR removing that hyperlink anyplace it appears in the docs would probably be a good plan.

**Reply by Krishna2864** [reply]
Thank you for highlighting that. I agree that removing the hyperlink from the documentation is a good step. if you confirm I'll take care of this and push a fix shortly

**Reply by lovelydinosaur** [reply]
Yup, please.

**Reply by Krishna2864** [reply]
here is the [PR](https://github.com/encode/django-rest-framework/pull/9174)
Thankyou @tomchristie for your support and feedback
